src/Util/Logger/Monolog/FriendicaDevelopHandler.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Friendica\Util\Logger;
+namespace Friendica\Util\Logger\Monolog;
```

Comment:
I can't comment on the relevant line. but why did you include `Friendica` in the class name? It already is in the `Friendica\Util\Logger\Monolog` namespace.